src/include/Jerryscript/VirtualMachine.h

```diff
@@ -4,29 +4,30 @@
  * http://github.com/SmingHub/Sming
  * All files of the Sming Core are provided under the LGPL v3 license.
  *
- * Jsvm.h
+ * VirtualMachine.h
```

Comment:
I'm not sure we really need a `VirtualMachine` class at all since it contains no state. We could instead place these methods as regular functions.

Reply:
> Let me know what you think , can submit another PR if you agree?
> .. We could instead place these methods as regular functions.

Ok, let's have it this way.